proposals/PROPOSALS.md

```diff
@@ -0,0 +1,142 @@
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+When the author and the HF Technical Track members agree on the final document, it is formally accepted: assigned a number, and scheduled for a final formal presentation.
+
+### Formal presentation (up to 1 month)
+
+During the next available Technical Track meeting, the author of the HIP presents the proposal to the audience. 
+
+### Merging the proposal
+
+If the HIP is accepted, the HF Technical Track will propose a release date, and the proposal will be merged directly by the HF Technical Track members.
+
+## Structure of the process
+
+The HIP process involves the following parties:
+
+1. The HIP Authors
+2. Relevant stakeholders affected by the change
+3. The HF Technical Track Leads
+
+### The HIP Authors
+
+Authors are responsible for building consensus within the community and documenting dissenting opinions before the HIP is officially discussed by the HIP HF Technical Track . Their goal is to convince the HF Technical Track that their proposal is useful and addresses pertinent problems in the Haskell ecosystem as well as interactions with already existing features. Authors can change over the lifecycle of the HIP.
+
+### The HF Technical Track
+
+The HF Technical Track is an experienced group of people with knowledge of the Haskell ecosystem, responsible for the strategic technical direction on behalf of the Haskell Foundation. Members are tasked with
+
+  - communicating with the community
+  - weighing in pros and cons of every proposal
+  - accepting, postponing or rejecting the proposal.
+
+HF Technical Track members should be either individuals responsible for a specific part of the Haskell ecosystem, or contributors and committers to parts of it. The members are selected by the HF CTO based on their expertise and implication in the community.
+
+The current HF Technical Track members are:
+
+- Emily Pillmore ([@emilypi](https://github.com/emilypi)), Haskell Foundation
+- Richard Eisenberg ([@goldfirere](https://github.com/goldfirere)), Tweag
+- Michael Snoyman ([@snoyberg](https://github.com/snoyberg)), FPComplete
+- Andrew Lelechenko ([@Bodigrim](https://github.com/Bodigrim)), Barclays
+- Davean Scies ([@davean](https://github.com/davean)), XKCD
+- Edward Kmett ([@ekmett](https://github.com/ekmett)), MIRI
+- Theophile Choutri ([@Kleidukos](https://github.com/Kleidukos)), Scrive
+- Gil Mizrahi (@soupi)
+
+### Voting
+
+For a HIP to be accepted, it must fulfill two requirements:
+
+- 66% of the HF Technical Track votes in favor of it
+- The HIP specifies a proposal lead (e.g. an implementor, or project director) who will represent the work done on the project.
```

Comment:
This outlines the votes for moving from 'Under Revision / Under Review' to 'Accepted'. I've assumed that moving from 'Submitted' to 'Under Review / Postponed' can be done by any individual HF Technical Track member (and made a suggestion for it). My reasoning to reject as a default is that this is the only outcome (except dormant) that does not require a write-up.

Note that there is now considerably duplication between this section and the end of the 'Formal Evaluation' section. It might be better to merge this into the formal evaluation section, where it is most relevant.

```suggestion
### Voting

When a HIP is scheduled at an HF Technical Track meeting (i.e. it is in 'Under Review'), it can be held to a vote for one of the following outcomes (in case it makes a difference: the HIP will be marked according to the first on this list to have a majority):

- Accepted (needs 66% of the HF Technical Track to vote in favor, and the HIP must specify a proposal lead, e.g. an implementor or project director, from the HF Technical Track who will represent the work done on the project.)
- Dormant (needs a simple majority, an HF Technical Track member that voted in favor will close the issue and PR and mark it as Dormant)
- Postponed (needs a simple majority, an HF Technical Track member that voted to postpone will close the issue and PR, and write clear conditions for reopening)
- Revision needed (needs a simple majority. This can only be the outcome of the vote four times, for a total of five rounds. An HF Technical Track member will write up what revisions are needed.)

If none of these applies, the proposal is Rejected.
```